content/articles/what-is-csr.markdown

```diff
@@ -12,7 +12,7 @@ The **Certificate Signing Request** (also **CSR** or **certification request**)
 The CSR contains information that will be included in your certificate, such as the [common name](/articles/what-is-common-name) and the company/owner details. It also contains the public key that will be embedded in the certificate.
 
 <info>
-We automatically generate the CSR for you using the information from the contact associated with the certificate along with an unique public/private encryption key pair. This works for the majority of cases, including hosting platforms such as Heroku or Amazon.
+We automatically generate the CSR for you using an unique public/private encryption key pair. This works for the majority of cases, including hosting platforms such as Heroku or Amazon.
```

Comment:
```suggestion
We automatically generate the CSR for you using a unique public/private encryption key pair. This works for the majority of cases, including hosting platforms such as Heroku or Amazon.
```

For some reason this sounded wrong to me (even as a 🇪🇸). Funny enough it was highlighted as a spelling error. [I also looked it up online](https://livexp.com/blog/a-unique-or-an-unique-which-article-is-correct-in-this-case/)

Reply:
Thanks! I agree :)